src/Tizen.NUI/src/public/Adaptor.cs

```diff
@@ -38,6 +38,8 @@ namespace Tizen.NUI
     ///
     /// </summary>
     /// <since_tizen> 4 </since_tizen>
+    [Obsolete("Deprecated in API8, will be removed in API10. This is not used anymore, please do not use.")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Please DO NOT hide the API that you want to deprecate. It should remain visible until its removal.